src/components/Layout.vue

```diff
@@ -64,12 +64,12 @@
 				if ( this.$store.state.matchInfo.redTeamName == undefined || this.$store.state.matchInfo.blueTeamName == undefined ) return;
 				fetch(`/wp-json/api/v1/public/data?data=teams&team=${this.$store.getters.redTeamName}`).then(async (redTeam)=>{
 					let redJson = await redTeam.json();
-					this.redTeam = redJson.data;
+					this.redTeam = !!redJson.data ? redJson.data : {};
 				});
 				
 				fetch(`/wp-json/api/v1/public/data?data=teams&team=${this.$store.getters.blueTeamName}`).then(async (blueTeam)=>{
 					let blueJson = await blueTeam.json();
-					this.blueTeam = blueJson.data;
+					this.blueTeam = !!blueJson.data ? blueJson.data : {};
```

Comment:
Sometimes the data is undefined, so these two lines check if the data is not null or undefined, otherwise it assign an empty object. redTeam and blueTeam are assigned to the team component and it expects team property to be an object type

Reply:
The double !! is a double negation. Its a handy and fast way of getting a boolean value. 
In js if something is null, false, undefiend, NaN, "" -0 or 0 in boolean context its considered false (falsy) and its double negation give you true if it is not falsy value

Reply:
Thats really cool :P